website/blog/2023-11-30-automated-security-versus-the-security-mindset/item.md

```diff
@@ -0,0 +1,163 @@
+
+Examples of these include:
+
+-   Scheduled tasks such as vulnerability scanning
+-   24/7 user and other activity monitoring
+-   Actions that require speed such as detecting and immediately responding to malicious intrusions
+
+Removing tasks like these from the manual operations of your SOC (security operations center) aids efficiency, supports your security team, and helps ameliorate any skills shortage.
+
+What are the benefits of an automated security system?
+
+Automated security also excels in:
+
+-   Reducing human error
+-   Eliminating manual steps
+-   Lowering the number of false positives
+-   Updating software
+-   Helping with compliance
+-   Enhancing incident response and threat intelligence
+
+## Why automation is a threat to cybersecurity
+
+If automation is such a popular and necessary asset in the cybersecurity field, why can't we automate everything?
+
+_Let’s think: Could over-reliance on automated security testing ultimately prove detrimental to cybersecurity and threaten the safety of your systems?_
+
+To help avoid this, we need to acknowledge that automation can't:
+
+-   Keep security teams up to date with new standards, such as the NIST Cybersecurity Framework; the ISO/IEC 27001 standard for information security management; the CIS Critical Security Controls; the OSSTMM; the Web Application Security Consortium (WASC 2.0); or the finance standard of PCI Data Security Standards for the payment card industry
+-   Adjust your internal security policies and practices to all the nuances of relevant industry, country or regulations such as NIST SP 800-52; The California Consumer Privacy Act: the Canadian PIPEDA; the EU’s GDPR; or HIPAA’s personal health data legislation
+-   Rapidly respond to every new CVE or every item that makes an appearance in the SANS Top 25, or the most common vulnerabilities listed in the OWASP lists
+-   Ensure that your own internal cybersecurity protocols and policies are enforced
+
+_But what else?_
+
+The first point to remember is that automated solutions can only reliably alert and respond to the threats to your network, services, databases, APIs, and applications that they've been configured to detect. This configuration is limited to the settings available in the particular software. Auto security testing tools are only as good as the rules that human security engineers have given them. They still must be configured and employed accurately.
+
+And, your own company’s internal business logic must be factored in. This is where pentesters (who may, of course, rely on some automated tools to help them identify some vulnerabilities across your network) can delve deep on specific vulnerabilities and apply your company’s custom business logic and data breach implications. Resultant summary reports must explain the business, financial, reputational, data, and user implications of likely breaches, investigations and penalties.
```

Comment:
Suggestion: "Automated tools cannot always take into account your company's own internal business processes." 

Dunno if you need that parenthetical.  You're just repeating that attackers have access to a wide array of tools, to which your pentesters hopefully have broad access.